contracts/button/errors.rs

```diff
@@ -0,0 +1,87 @@
+use ink_env::Error as InkEnvError;
+use ink_prelude::{format, string::String};
+use openbrush::contracts::psp22::PSP22Error;
+
+/// GameError types
+#[derive(Debug, PartialEq, Eq, scale::Encode, scale::Decode)]
+#[cfg_attr(feature = "std", derive(scale_info::TypeInfo))]
+pub enum GameError {
+    /// Returned if reset is called before the deadline
+    BeforeDeadline,
+    /// Returned if button is pressed after the deadline
+    AfterDeadline,
+    /// Returned if a call is made from an account with missing access control privileges
+    MissingRole(String),
```

Comment:
this could just wrap the Role, enum, i.e. `MissingRole(Role)`

Reply:
done